content/docs/studio/get-started.md

```diff
@@ -27,116 +22,85 @@ details]).
 
 </admon>
 
-## Explore demo project
-
-1. If this is the first time you are signing in to Iterative Studio, you will
-   see that there already exists a `Demo` project that connects to an example
-   repository. Use this project to explore the features that Iterative Studio
-   has to offer.
+## Run and track experiments
 
-![](https://static.iterative.ai/img/studio/login_home_v3.png) _When you first
-login, an example project is already created for you to explore, and you can add
-more projects._
+1. Iterative Studio works with Git repositories. So first, [prepare your Git
+   repositories] and make sure that
+   [the connection to your Git server has been set up](/doc/studio/user-guide/account-management#git-integrations).
 
-2. Each project on this dashboard displays the metrics that Iterative Studio
-   identified in your Git repository. In the figure above, you can see that
-   `avg_prec` and `roc_auc` metrics are displayed.
+[prepare your git repositories]:
+  /doc/studio/user-guide/projects-and-experiments/what-is-a-project#prepare-your-repositories
 
-## Add your projects
+2. Click on `Add a Project` and follow the on-screen instructions to search and
+   connect to the desired Git repository. You can `Skip and Continue` the
+   `Project settings`.
 
-1. To connect to a new repository, click on `Add a Project`. All the
-   organizations that you have access to will be listed.
+   <admon type="info">
 
-    <admon type="info">
+   Once the project is created, remember to [configure project
+   settings][project settings] if [needed][when do you need project settings].
 
-   If you do not see your desired organizations or Git repositories, make sure
-   that
-   [the connection to your Git server has been set up](/doc/studio/user-guide/account-management#git-integrations).
+   </admon>
 
-   To connect to your GitHub repositories, you must install the Iterative Studio
-   GitHub app. Refer to the section on
-   [GitHub app installation](/doc/studio/user-guide/install-github-app) for more
-   details.
+3. Iterative Studio parses your project's `dvc.yaml` file to identify data,
```

Comment:
@amritghimire I've written this in the docs now: `Iterative Studio parses your project's dvc.yaml file to identify data, metrics, plots and hyperparameters.`. Can you pls confirm if it is technically correct?

Reply:
It may not always be technically correct. It does use the dvc.yaml file to identify the data, metrics, plots and hyper parameters. But sometime we parse params or metrics user manually select as well.

Reply:
> But sometime we parse params or metrics user manually select as well.

This is only in the case of custom files being specified, right?

Reply:
So it's correct if the docs mention that except in the case of custom files, the repos should have a `dvc.yaml` file with the metrics / params / plots that users want to show in Studio. Right?

Reply:
Yes.

Reply:
Thanks @amritghimire!